headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -1023,13 +1039,18 @@ struct dungeon_generation_info {
     // 0x0: Set if the floor layout is guaranteed to be a Monster House, or the dungeon generation
     // algorithm fails
     bool force_create_monster_house;
-    undefined field_0x1;
-    undefined field_0x2;
+    // 0x1: Set if the locked door on the dungeon floor has already been open.
```

Comment:
has already been open**ed**? And is this referring to a prior visit to the dungeon? Otherwise I'm not sure why this would be set on the dungeon generation info at the start of a floor.

Reply:
The function (0x023439D8 NA) that actually opens the locked door that's called by the ApplyKeyEffect specifically checks if the value is 0, if it is it will try to open the door and will set the value to 1. When it's 1, it just logs a can't be used message without checking. It's also initialized to 0 in GenerateFloor.

Since it's initialized to 0 in GenerateFloor I'm pretty sure it's a tracker on a by floor basis.

Reply:
Okay, let's just fix the typo then.

Reply:
This still needs to be fixed

Reply:
I fixed this one.